interactive_engine/src/executor/gaia_runtime/src/server/manager.rs

```diff
@@ -218,7 +218,7 @@ fn parse_store_ip_list_for_gaia(address_list: &[RuntimeAddressProto]) -> Vec<Pee
             server_id: server_idx,
             ip: address.get_ip().to_string(),
             // assign a random port for pegasus
-            port: 0,
+            port: 8198,
```

Comment:
Fixed port of multiple instances will conflict in local mode?

Reply:
maybe we should have a config file to specify the port?